specs/experimental/zk-fault-dipute-game/overview.md

```diff
@@ -13,14 +13,99 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+[g-bisection]: ../../../specs/fault-proof/challenge.md#bisection
+[g-challenge-creation]: ../../../specs/fault-proof/challenge.md#challenge-creation
```

Comment:
These are not glossary. Please rename these to `[prev-bisection]`, `[prev-challenge-creation]`. Also move these below each corresponding usage section.